mods/default/locale/con.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
```

Comment:
While such scripts may be useful for your translation, please don't commit them to the repository.

Reply:
Extra files have been deleted.